example.config.yml

```diff
@@ -271,3 +271,4 @@ selenium_version: 2.46.0
 # Other configuration.
 dashboard_install_dir: /var/www/dashboard
 known_hosts_path: ~/.ssh/known_hosts
+ssh_home: "{{ drupal_core_path }}"
```

Comment:
Might the word "home" be confused with the path `/home/`? Perhaps this should be `ssh_dir` or `ssh_path` or `ssh_dest`?